pytest/conftest.py

```diff
@@ -0,0 +1,90 @@
+import json
+import uuid
+
+import pytest
+from django.contrib.auth.models import User
+
+from hs_access_control.models import UserAccess
+from hs_core import hydroshare
+from hs_labels.models import UserLabels
+
+
+@pytest.mark.django_db
+@pytest.fixture(scope="function")
+def resource_with_metadata():
+    """Resource with metadata for testing"""
+    rtype = 'GenericResource'
+    res_uuid = str(uuid.uuid4())
+    title = 'Resource {}'.format(res_uuid)
+    metadata = []
+    metadata.append({'coverage': {'type': 'period', 'value': {'start': '01/01/2000',
```

Comment:
I had a version with the dictionary created all at once, but this is more readable/maintainable if people want to make changes. Hence the various sections, with metadata.append